modules/apps/frontend-theme/frontend-theme-classic/src/css/_clay_variables.scss

```diff
@@ -12,5 +12,42 @@ $btn-outline-secondary: (
 	color: #6b6c7e,
 );
 
+$btn-success: (
+	enabled: false,
+);
```

Comment:
not sure why this isn't working, the purpose is remove the unused buttons

Reply:
It might be due to custom properties? I tested on Clay CSS seems to be working. The way to find out is to check the clay.css file in the head tag and search for `.btn-success`, `.btn-info`, etc.

Reply:
I wasn't loading the file 🤦